designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/ImageBlock.kt

```diff
@@ -55,6 +63,17 @@ fun <T> ImageBlock(
     }
 
     if (image != null) {
+        if (isFullScreen) {
```

Comment:
This would cause a weird issue where the image block is removed from the list entirely and only full screen image is shown. You can see it in the sample app, when you click on the image, the image block is removed and then the full screen image is shown. 

I don't think that's the right behaviour, can you check with Marcos once and confirm if any changes are required?